src/sympc/utils/mpc_utils.py

```diff
@@ -0,0 +1,31 @@
+from typing import List
+
+from sympc.tensor import ShareTensor
+
+# TODO: Remove this
+import torch
+
+
+def count_wraps(share_list: List[torch.tensor]) -> torch.Tensor:
+    """Count the number of overflows and underflows that would happen if we
+    reconstruct the original value
+
+    This function is taken from the CrypTen project.
+    CrypTen repository: https://github.com/facebookresearch/CrypTen
+
+    Args:
+        share_list (List[ShareTensor]): List of the shares
+
+    Returns:
+        The number of wraparounds
+    """
+
+    res = torch.zeros(size=share_list[0].size(), dtype=torch.long)
+    prev_share = share_list[0]
+    for cur_share in share_list[1:]:
+        next_share = cur_share + prev_share
+        res += ((prev_share < 0) & (cur_share > 0) & (next_share > 0)).long()
```

Comment:
You have a typo here!
https://github.com/facebookresearch/CrypTen/blob/81ae3df7f2025ec727e8012fb0703dd91da0e079/crypten/common/util.py#L36